ui/raidboss/data/02-arr/dungeon/pharos_sirius.ts

```diff
@@ -0,0 +1,54 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.PharosSirius,
+  triggers: [
+    {
+      id: 'Pharos Sirius Deathly Cadenza',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5CF', source: 'Siren', capture: false }),
+      response: Responses.getIn(),
+    },
+    {
+      id: 'Pharos Sirius Feral Lunge',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5CC', source: 'Siren', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'Pharos Sirius Corrupted Crystal',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '176', count: '03' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get Away : Stacks Explode Soon',
```

Comment:
```suggestion
          en: 'Spread: Stacks Explode Soon',
```

I think this is "spread" vs "get away (from the boss)"

Reply:
it's a "get away from party", really. will change that

Reply:
Yeah, mostly just in a cactbot consistency sense.  Usually spread = get away from other people and get away = stuff is chasing you or get away from boss or a specific thing.

Reply:
I see, that's fair. will change it ^^